src/runtime_src/core/tools/common/tests/aie_pl_util/pl_controller_aie2.cpp

```diff
@@ -122,27 +121,6 @@ void plController_aie2::enqueue_loop_end() {
     m_opcodeBuffer.add(CMD_TYPE::LOOP_END);
 }
 
-void plController_aie2::enqueue_set_and_enqueue_dma_bd(const std::string& portName, int idx, int dma_bd_len, int id) {
```

Comment:
removed the plController_aie2::enqueue_set_and_enqueue_dma_bd() since its not being used on aie2 pl test. Another reason it calls get_buffers() which uses reference to dma_lock_report.json file. On aie2 test there is no reference to this json file.